site/content/entry/yuzu-mini-dev-2/index.md

```diff
@@ -0,0 +1,133 @@
+Blinkhawk helped by cleaning up my bugs to get the thing working and he's done an awesome job.
+A generic texture cache allows us to share the complex part of it across graphics APIs.
+That means that we will have the same overall logic for OpenGL and Vulkan, making it easier to maintain and improve.
+
+It's not as good as Dolphin's VideoCommon, one of the better examples for generic graphics code, but it's still better than having to copy-paste code everywhere.
+This is one of the last pre-requirements for the Vulkan API.
+
+I've also been working on implementing various compute shaders related instructions like shared memory (ARB_compute_shader), atomic operations, memory shuffles (NV_shader_thread_shuffle), votes (NV_gpu_shader5, NV_shader_thread_group), surface operations (SUATOM, SULD).
+These are all features from the D3D11/12 era, meaning that we are reaching interesting complexity on the shaders we find in recently released games.
+I am also investigating how to implement some missing functionality in yuzu, but that's a story for another time.
+
+**Q: Would you like to say something to our audience?**
+
+**R:** Well, there are three things I'd like to say.
+
+`First and foremost...` most of our GPU work has been possible thanks to research done by the people working on nouveau (mesa's Nvidia free driver) and libnx's nouveau port to Switch. They have been of huge help.
+
+`Second...` many people believe that Vulkan will bring lots of performance improvements to the emulator, I was one of those too. Don't get me wrong, it might bring some boost on some hardware vendors. 
+But the main issue resides in Vulkan's design. It is designed in a way that the programmer cooks and reuses its resources, but in emulation you can't easily know what's going to happen in the future. On the other hand, OpenGL by design works without backing its commands. 
+
+Some people might ask, why would we want two APIs? I think one API is going to perform better on two vendors, while the other API is prevalent on the other vendor. 
+
+`And last but not the least...` We have a console that's just two years old and it's still lacking the mainstream entries for most Nintendo IPs: Metroid, Donkey Kong Country, the announced Gen 8 Pokémon, Star Fox, Mario Kart, F-Zero (who knows? maybe they will find a way to refresh the franchise) and the end-of-cycle Zelda.
+
+I'm really excited about what's to come and how good Switch emulators will perform with these games. Emulation has been surprisingly faster in the last few years. Let's look forward to a wonderful future...
+***
+
+This has been absolutely wonderful.
+Huge thanks to [Rodrigo](https://github.com/ReinUsesLisp), for taking time off his busy schedule to give us a brief insight into his life.
+We wish you success in all your future endeavours. Keep making yuzu awesome!
+
+We hope you guys enjoyed it, and we will be back soon with another featured Yuzu dev to bring you guys behind the curtain again. 
```

Comment:
Remove gendered wording:
`We hope you all enjoyed` and `bring you behind`

Also, `Yuzu` to `yuzu`